query/query.go

```diff
@@ -156,6 +156,83 @@ func Contains(attribute string, value interface{}) string {
 	})
 }
 
+func NotContains(attribute string, value interface{}) string {
```

Comment:
Come to think of it, should value here be array of values? 🤔 Maybe we could make this better, irrelevant of 1.8.x, just small improvement